libs/template/labs/src/i18n/translations/en.ts

```diff
@@ -35,7 +35,9 @@ const order = {
 
 const user = {
   'user.address.remove-info':
-    'Removing this address will not remove any pending orders being dispatched to this address',
+    'Removing this address will not remove any pending orders being dispatched to this` address',
+  'user.login.benefits-of-registering-to-the-website':
```

Comment:
But are we using this key anywhere?

Reply:
it is leftover from prev implementation. Removed it